modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Button.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+export default function Button({
+	componentId: _componentId,
```

Comment:
Do we need to define here all this props (apart from `label`)?

Reply:
Well, that's a good question @carloslancha 

I had a look at the [`DropDownMenu`](https://github.com/liferay/liferay-portal/blob/8de11d8e9d6424a9dc9ae8f371839e58263dc111/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js#L22-L33) and decided to use the same "pattern"

I believe that if we don't do this, we will get this kind of error 

![error01](https://user-images.githubusercontent.com/5572/91998232-41bd4d80-ed3b-11ea-8076-ecd5f3b82c9c.png)


But @jbalsas can probably confirm